pkgs/applications/science/programming/scyther/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, callPackage, callPackage_i686, buildEnv
+, includeGUI ? true
```

Comment:
Thinking about it again, it might be better to default to false, the current NixOS target audience would probably prefer CLI tools (me included). Also the CLI tool doesn't have the mentioned runtime problems with certain protocol